docs/excel/make-custom-functions-compatible-with-xll-udf.md

```diff
@@ -44,15 +41,15 @@ The following example shows how to specify both a COM add-in and an XLL as equiv
 ```
 
 > [!NOTE]
-> If an add-in declares its custom functions to be XLL compatible, changing the manifest at a later time could break a user's workbook because it will change the file format.
+> If an Excel JavaScript API add-in declares its custom functions to be compatible with an XLL add-in, changing the manifest at a later time could break a user's workbook because it will change the file format.
```

Comment:
This paragraph could use some elaboration. Exactly what changes in the manifest would have this effect? Does this mean that once an add-in declares its custom functions to be compatible with an XLL, it can never reverse this declaration? In what sense of format is the workbook's file format changed?